build/yamls/base/conf/antrea-agent.conf

```diff
@@ -5,6 +5,10 @@ featureGates:
 # Service traffic.
 #  AntreaProxy: true
 
+# Enable full Service support in AntreaProxy in antrea-agent. Without KubeProxy, NodePort/LoadBalancer
+# can be accessed from outside the cluster, and ClusterIP can be accessed from host.
```

Comment:
NodePort/LoadBalancer Services can be accessed from inside the cluster too. So should we just say "NodePort/LoadBalancer Services are supported, and ..."

Reply:
Agreed.